shared/settings/contacts-joined/index.native.tsx

```diff
@@ -2,41 +2,46 @@ import * as React from 'react'
 import * as Container from '../../util/container'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
+import * as RPCTypes from '../../constants/types/rpc-gen'
 import {FollowButton, WaveButton} from './buttons'
+import * as Tracker2Constants from '../../constants/tracker2'
+type Props = {}
 
-type ResolvedContactEntry = {
-  username: string
-  contactLabel: string
-}
-type Props = Container.RouteProps<{
-  people: Array<ResolvedContactEntry>
-}>
-
-const renderItem = (_: number, {username, contactLabel}: ResolvedContactEntry) => (
-  <Kb.Box2 direction="horizontal" key={username} fullWidth={true}>
-    <Kb.Box style={styles.avatar}>
-      <Kb.Avatar username={username} size={48} />
-    </Kb.Box>
-    <Kb.Box2 direction="vertical" style={styles.rightBox}>
-      <Kb.ConnectedUsernames colorFollowing={true} type="BodySemibold" usernames={[username]} />
-      <Kb.Text type="BodySmall">{contactLabel}</Kb.Text>
-      <Kb.Box2 direction="horizontal" gap="tiny" fullWidth={true} style={styles.buttons}>
-        <FollowButton username={username} small={true} />
-        <WaveButton usernames={username} small={true} />
+const renderItem = (_: number, item: RPCTypes.ProcessedContact) => {
+  const username = item.username
+  const label =
+    item.contactName || (item.component && (item.component.phoneNumber || item.component.email)) || ''
+  const followThem = Container.useSelector(state => Tracker2Constants.followThem(state, username))
```

Comment:
This isn't a component, can't call hooks here. I get a red screen on enabling contact import.